cad/scripts/test_drawing_layout_check.py

```
@@ -1538,3 +1600,25 @@ def test_a_balloon_whose_item_reads_empty_does_not_tie_every_key():
+            self.revision = revision
+
+        def GetCustomInfoValue(self, _configuration, name):
+            return self.revision if name == "Revision" else "present"
+
+    expected = drawing_common._config.release_revision()
+    drawing_common.read_required_properties(
+        Model(expected),
+        ("Revision",),
+        required=("Revision",),
+    )
+    with pytest.raises(RuntimeError, match="Revision .* current release"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)